src/sentry/api/endpoints/sentry_app_installation_details.py

```diff
@@ -23,5 +23,9 @@ def delete(self, request, installation):
                             actor=request.user):
             return Response(status=404)
 
-        Destroyer.run(install=installation)
+        Destroyer.run(
+            install=installation,
+            user=request.user,
+            action='deleted',
```

Comment:
Doesn't look like the Mediator takes an `action` param. I feel like this should be failing something...